modules/Kconfig.tls-generic

```diff
@@ -6,6 +6,7 @@
 
 menu "TLS configuration"
 
+if MBEDTLS
```

Comment:
Could put  a blank line after `if` and a blank line before `endif`. That's consistent with https://docs.zephyrproject.org/latest/guides/kconfig/index.html#header-comments-and-other-nits now too.

Reply:
done